vmware_dvs/utils/dvs_util.py

```diff
@@ -25,24 +25,33 @@
 
 from vmware_dvs.common import constants as dvs_const
 from vmware_dvs.common import exceptions
+from vmware_dvs.utils import spec_builder
 
 LOG = log.getLogger(__name__)
 
+CREATING_PG_STATUS = 'creating'
+READY_PG_STATUS = 'ready'
+UPDATING_PG_STATUS = 'updating'
+REMOVING_PG_STATUS = 'removing'
+
+INIT_PG_PORTS_COUNT = 4
+
+# TODO(ekosareva): get task_pool_interval from config
+TASK_POOL_INTERVAL = 2
```

Comment:
too long. We need to return port key as soon as possible. Agent will do it too slowly if customer wants many VMs at once at one PG.